pkg/apb/executor.go

```diff
@@ -80,27 +81,28 @@ func ExecuteApb(
 	if err != nil {
 		return executionContext, err
 	}
-	if len(secrets) > 0 {
-		executionContext.Namespace = clusterConfig.Namespace
-		executionContext.Targets = append(executionContext.Targets, context.Namespace)
-	} else {
-		// Using a new UUID is sane, because it will have some gurantee
-		// of uniquenes and will meet DNS name requirements.
-		executionContext.Namespace = uuid.New()
-		executionContext.Targets = append(executionContext.Targets, context.Namespace)
-		// Create namespace.
-		namespace := v1.Namespace{
-			ObjectMeta: metav1.ObjectMeta{
-				Labels: map[string]string{transientNameSpaceKey: spec.FQName},
-				Name:   executionContext.Namespace,
-			},
-		}
-		_, err := k8scli.CoreV1().Namespaces().Create(&namespace)
-		if err != nil {
-			return executionContext, err
-		}
+
+	// Using a new UUID is sane, because it will have some gurantee
+	// of uniquenes and will meet DNS name requirements.
```

Comment:
nit

Using a new UUID is sane, because it will have some **guarantee**
of **uniqueness** and will meet DNS name requirements.